TigranKirakosov/functions.js

```diff
@@ -0,0 +1,66 @@
+// 1) splitAndMerge
+
+//version 1
+function splitAndMerge(str, sp) {
+  let acc = [];
+  let newStr = '';
+  for (let i = 0; i < str.length; i++) {
+    if (str[i] === ' ') {
+      acc[i] = str[i];
+    } else {
+      if (str[i + 1] === ' ' || i === str.length - 1) {
+        acc[i] = str[i];
+      } else {
+        acc[i] = str[i] + sp;
+      }
+    }
+    newStr += acc[i];
+  }
+  return newStr;
+}
+
+//version 2
+function splitAndMerge2(str, sp) {
+  let newStr = '';
+  str.split(' ').forEach((word, index, arr) => {
+    index === arr.length - 1
+      ? (newStr += word.split('').join(sp))
+      : (newStr += word.split('').join(sp) + ' ');
+  });
+  return newStr;
+}
+
+// 2) hashToArr
+function hashToArr(hash) {
+  return Object.keys(hash).map(prop => [prop, hash[prop]]);
+}
+
+// 3) toCamelCase
+function toCamelCase(str) {
+  let newStr = '';
+  let acc;
+  if (str.includes('-')) acc = str.split('-');

```

Comment:
``String.prototype.includes` is a part of ES6/ES2015 http://www.ecma-international.org/ecma-262/6.0/#sec-string.prototype.includes

Reply:
That's sad news :cry: